tests/sentry/utils/hashlib/tests.py

```diff
@@ -17,7 +17,12 @@
     ("seed", six.binary_type("test".encode("utf-8")), "334e3fd2f66966a5c785d825c5f03494"),
     ("seed", six.text_type("test"), "ce35c0ce0d38976f61a5ca951de74a16"),
     ("seed", (4, 2), "d03b32e798444249d726158594d370f6"),
-    ("seed", {"test": 42}, "c545cf1c4ab09eff4a1e0fa5209f1645"),
+    ("seed", {six.text_type("test"): 42}, "ca094da15d323155e3954cff7ca373c4"),
```

Comment:
Can we not do `"test".encode("utf-8"): 42` instead and get the same hash?

Reply:
Yeah true I could have done that, but I think this case makes a bit more sense. It doesn't invalidate my comment below this :(

Reply:
Oh, I'm not saying your comment is not valid 😀 This was just to avoid cache key changes between python versions.

Reply:
oh, yeah maybe if we went through and verified all cache keys were binary, we could avoid it :)